WordPress/src/main/java/org/wordpress/android/ui/stories/prefs/StoriesPrefs.kt

```diff
@@ -127,7 +206,7 @@ class StoriesPrefs @Inject constructor(
         }
     }
 
-    fun replaceLocalMediaIdKeyedSlideWithRemoteMediaIdKeyedSlide(
+    fun replaceLocalMediaIdKeyedSlideWithRemoteMediaIdKeyedSlide_Phase2(
```

Comment:
could we remove the underscore from this method name and the one below since function names shouldn't contain them? Also could you expound a bit on why we need the `phase1` and `phase2` in the names? Is there anyway we can make it clearer? Probably some comments here? LMK

Reply:
Excellent idea, I didn't know how to enforce that enough so I thought calling the attention of the reader by making that underscore postfix name on purpose would make it "special enough" but ofc that goes with moving away from standards, which we don't want 👍 
Addressed as suggested in 9b80af491a "renamed _Phase1 and _Phase2 postfixed method names and added comments to illustrate how id assignment for media works in two phases"

Reply:
Those comments are beautifully done. Everything is all clear now 🤝